ivadomed/training.py

```diff
@@ -283,23 +316,32 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
     if model_params["name"] == "FiLMedUnet" and debugging:
         save_film_params(gammas_dict, betas_dict, contrast_list, model_params["depth"], log_directory)
 
-    try:
```

Comment:
I am not sure why we need to change these lines

Reply:
If we don't have a validation dataset, then no checkpoint file, then no best model.